examples/README.md

```diff
@@ -12,9 +12,9 @@ This directory contains examples of Bokeh Apps, which are simple and easy to cre
 
 ### [`charts`](https://github.com/bokeh/bokeh/tree/master/examples/compat)
```

Comment:
I'd like to propose changing these to relative links. (e.g. https://github.com/bokeh/bokeh/commit/0d0b62b7aebbfa0f94180ac6de9e3d023e0eca94). That way, if examples change over time, people will be pointing to the correct branch.  I think this is particularly true, it we wanted to link to a release tagged copy of this README